missioncontrol_client/__init__.py

```diff
@@ -1,10 +1,83 @@
 import argparse
 import json
 import os
+
+# dl
+import io
+import zlib
+import socket
+import datetime
+from pyblake2 import blake2b
```

Comment:
Why not builtin? https://docs.python.org/3/library/hashlib.html#hashlib.blake2b

Reply:
I'm relying on planetlabs/datalake, which uses pyblake2

Reply:
I guess I could vendor this functionality actually and make the modifications directly.